osu.Game.Rulesets.Catch/osu.Game.Rulesets.Catch.csproj

```diff
@@ -15,4 +15,8 @@
   <ItemGroup Label="Project References">
     <ProjectReference Include="..\osu.Game\osu.Game.csproj" />
   </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Update="Microsoft.CodeAnalysis.BannedApiAnalyzers" Version="3.3.4" />
+  </ItemGroup>
```

Comment:
Not sure why this one is showing up in almost every project. Should likely be bumped via `Directory.Build.props` instead:

https://github.com/ppy/osu/blob/f15698d02535c3c8be7f2453c5dcffd0e9dda71b/Directory.Build.props#L19-L22